### PR TITLE
Adds a trait that removes your legs

### PR DESCRIPTION
**Pull request by muzcl**
## About the PR

Adds a new trait called 'Stumped', it removes both your legs on roundstart and gives you a wheelchair.

## Why's this needed?

Playing as a wheelchair bound character is hard to set up, since you have to go to medbay, get your legs taken off (or pay 5000 spacebux), and get a wheelchair. This makes it so theres a fast and easy way to do it that doesn't involve paying loads of spacebux.

## Changelog

```changelog
(u)Idiot
(+)Adds Stumped trait that spawns you with no legs and a free wheelchair.
(-)Removes No Legs spacebux purchase.
```

### CODE REVIEW

**Comment by Conchuckter** [comment]
Looks like you made your changes in your master branch, should work fine but might be tricky if you plan to make future changes.

**Comment by Sord213** [comment]
If you're removing the spacebux purchase it would be better to just remove it entirely rather than leave it commented out.

File: code/modules/economy/persistent_bank_purchases.dm [hunk 1]
```diff
@@ -434,26 +434,27 @@ var/global/list/persistent_bank_purchaseables =	list(\
...
+//				var/mob/living/carbon/human/H = M
+//				SPAWN(6 SECONDS)
+//					if (H.limbs)
+//						if (H.limbs.l_leg)
+//							H.limbs.l_leg.delete()
+//						if (H.limbs.r_leg)
+//							H.limbs.r_leg.delete()
+//						boutput( H, "<span class='notice'><b>You haven't got a leg to stand on!</b></span>" )
+//				return 1
+//			return 0
```
Comment (Conchuckter) [inline code/modules/economy/persistent_bank_purchases.dm:457]: If you are going to leave this in use multiline comments with /* and */

File: code/procs/jobprocs.dm [hunk 2]
```diff
@@ -686,6 +686,15 @@
...
 				boutput(src, "<b>Your singular arm makes you feel responsible for crimes you couldn't possibly have committed.</b>" )
 
+	if (src.traitHolder && src.traitHolder.hasTrait("nolegs"))
+		if (src.limbs)
+			SPAWN(6 SECONDS)
+				if (src.limbs.l_leg)
+					src.limbs.l_leg.delete()
+				if (src.limbs.r_leg)
+					src.limbs.r_leg.delete()
+			src.put_in_hand_or_drop(new /obj/stool/chair/comfy/wheelchair)
```
Comment (Conchuckter) [inline code/procs/jobprocs.dm:696]: I'd probably just set it to the mobs turf by doing:
`new /obj/stool/chair/comfy/wheelchair(get_turf(src))`

File: code/datums/traits.dm [hunk 3]
```diff
@@ -512,6 +512,13 @@
 	icon_state = "placeholder"
 	points = 0
 
+/datum/trait/nolegs
+	name = "Stumped"
+	desc = "Because of a freak accident involving a piano, a forklift, and lots of vodka, both of your legs had to be amputated. Fortunately, NT has kindly supplied you with a wheelchair (due to regulations) out of the goodness of their heart."
```
Comment (Conchuckter) [inline code/datums/traits.dm:517]: I think it sounds better if `(due to regulations)` is at the end.

**Review by Conchuckter — CHANGES_REQUESTED** [review]

File: code/procs/jobprocs.dm (hunk 2 above)
Comment (muzcl) [reply code/procs/jobprocs.dm:696]: Done,

File: code/datums/traits.dm (hunk 3 above)
Comment (muzcl) [reply code/datums/traits.dm:517]: and done.

**Pull request merged**